.github/workflows/update-boost-version.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
         run: |
           # CMake for extracting present boost version.
           mkdir build && cd build
-          cmake ..
+          cmake -DDOWNLOAD_DEPENDENCIES=ON ..
```

Comment:
Up to you---maybe installing `libstb-dev` and `libensmallen-dev` would be sufficient?

Reply:
Sure, that should work as well, changed it.